network/src/protocols/discovery/mod.rs

```diff
@@ -230,7 +225,7 @@ impl<M: AddressManager> ServiceProtocol for DiscoveryProtocol<M> {
                 state.send_messages(context, *id);
                 // check timer
                 state
-                    .check_timer(now, dynamic_query_cycle)
+                    .check_timer(now, ANNOUNCE_INTERVAL)
```

Comment:
Why `announce_check_interval` is not used here?

Reply:
`ANNOUNCE_INTERVAL` is setup as every 24 hours to send announce nodes message, it's a fixed value, and `announce_check_interval` is a configurable parameter to check whether 24 hours have elapsed.